.eslintrc

```diff
@@ -23,6 +24,8 @@
     "jsx-quotes"           : [2, "prefer-single"],
     "max-len"              : [2, 120, 2],
     "object-curly-spacing" : [2, "always"],
-    "comma-dangle"         : "off"
+    "comma-dangle"         : "off",
```

Comment:
setting 'comma-dangle' to 'off' means that eslint does not perform any checking for this rule. So currently you can have or ommit dangle comma and there will be no errors whatsoever.

Probably what you meant is `"comma-dangle"         : [2, "always-multiline"],`